### DISCUSSION TITLE
Gallery example for openwrt

### DISCUSSION THREAD

**Discussion by remys** [utmapp/UTM#4863] [UTM Help (macOS)]
Hi all,
Just discovering utm and QEMU. Nice work!
To play around with, I tried to create a virtual router in utm based on openwrt (also new stuff to me!), but no success.
Question: has any one tried this exercice, and if so, could you place this as an example in the gallery?

**Comment by joshenders** [comment]
I have an OpenWrt installation running in UTM. Are there any particular questions you have, if this is still an issue for you?

**Comment by joshenders** [comment]
For anybody finding this via a search engine, I've created a guide of sorts here: https://forum.openwrt.org/t/installing-openwrt-on-m1-mac-apple-silicon-virtualization-software/128553/7?u=joshenders
